docs/_static/site.css

```diff
@@ -23,3 +23,28 @@
 	height: 3rem;
 }
 
+:root {
+	--primary-accent: #4ba3d9;
+	--base-color: #f7f7f7;
+	--iver-yelow: #fcde06;
+	--mono-hue: 218;
+	--mono-saturation : 12%;
+	--base-background-color: #292c31;
+	--sidebar-name-color: var(--primary-accent);
+	--sidebar-nav-link-color--active: var(--primary-accent);
+	--code-theme-keyword: var(--primary-accent);
+	--sidebar-background: #212122;
+	--mark-background:  var(--iver-yelow);
+	--code-theme-variable: var(--iver-yelow);
+	--code-theme-punctuation: var(--base-color);
+	--code-theme-text: var(--base-color);
+	--code-theme-selector: var(--iver-yelow)
+}
+
+.app-name-link {
+	font-size: 37px;
```

Comment:
maybe you can use rem instead of px here so it can scale up and down.

Reply:
Of course, something like this?

![image](https://user-images.githubusercontent.com/22037089/111813556-a1df9f80-88d9-11eb-9dae-34c7782dd569.png)

![image](https://user-images.githubusercontent.com/22037089/111813675-cb98c680-88d9-11eb-9a0f-79daf8174ecd.png)

![image](https://user-images.githubusercontent.com/22037089/111813718-d94e4c00-88d9-11eb-89dc-13033e5892a9.png)

Reply:
Looks good to me! But got to ask, where can I find where this CSS class is applied? Might be blind but I can't find it when I look around the files. I just noted that it was a link because of the naming so that's why I thought it should be in rem instead of px.

Reply:
A lot of the classes and stuff are generated by docsify. This is not just a static website. To view this locally I needed to install the docsify cli `npm i docsify-cli -g` and then run `docsify serve` under `/docs`:

![image](https://user-images.githubusercontent.com/22037089/111823095-d442ca00-88e4-11eb-9a41-06ef0d02cfd7.png)

Reply:
ah, I see. resolving.